lambda/query-user-services/query-user-services.ts

```diff
@@ -71,7 +71,7 @@ const createUserRecordEvent = (
 };
 
 export const sendSqsMessage = async (
-  messageBody: UserRecordEvent,
+  messageBody: object,
```

Comment:
Interesting, what caused the shift from a specific interface to `object`? I'm assuming `object` is just a generic for any `{}`.

Ideally I'd like to keep these specific?